scheduler/src/cook/components.clj

```diff
@@ -399,6 +416,8 @@
                    role)
      :mesos-framework-name (fnk [[:config [:mesos {framework-name "Cook"}]]]
                              framework-name)
+     :mesos-framework-id (fnk [[:config [:mesos {framework-id nil}]]]
```

Comment:
`:mesos-framework-id` resides in the `config-settings` component and represents the value that cook was configured with. `:framework-id` resides in `scheduler-server`, it consumes `:mesos-framework-id`, and it handles the defaulting to the ZK value if nothing was configured.